google/cloud/spanner/samples/postgresql_samples.cc

```diff
@@ -557,28 +559,106 @@ void InformationSchema(
 }
 // [END spanner_postgresql_information_schema]
 
-/*
- * Remaining samples to add:
- *   - Insert a Venue with a valid for the Revenues column.
- *   - Insert a Venue with a NULL value for the Revenues column.
- *   - Insert a Venue with a NaN value for the Revenues column.
- *   - Mutations can also be used to insert/update NUMERIC values,
- *     including NaN values.
- */
 // [START spanner_postgresql_numeric_data_type]
 void NumericDataType(google::cloud::spanner::Client client) {
-  // Get all Venues and inspect the Revenues values.
+  // Insert a Venue with a valid value for the Revenue column.
+  google::cloud::spanner::DmlResult dml_result;
+  auto commit = client.Commit(
```

Comment:
I think this example would benefit from `namespace spanner = ::google::cloud::spanner`.

Reply:
I originally thought that it might benefit from not having them, but we do use such an alias in the non-postgres samples, so I'll revisit.  Thanks.